airbyte-webapp/src/components/EntityTable/ConnectionTable.tsx

```diff
@@ -189,9 +173,44 @@ const ConnectionTable: React.FC<ConnectionTableProps> = ({ data, entity, onClick
         },
         cell: (props) => <ConnectionSettingsCell id={props.cell.getValue()} />,
       }),
-    ],
-    [columnHelper, sortBy, sortOrder, onSortClick, entity, allowAutoDetectSchema]
-  );
+    ];
+
+    if (streamCentricUIEnabled) {
+      columns.unshift(
+        columnHelper.display({
+          id: "stream-status",
+          cell: StreamsStatusCell,
```

Comment:
The cell doesn't have a fixed width, so it gets longer as the table resizes

Reply:
Added `size: 170,`